doc/modules/linear_model.rst

```diff
@@ -1019,6 +1019,62 @@ considering only a random subset of all possible combinations.
 
     .. [#f2] T. Kärkkäinen and S. Äyrämö: `On Computation of Spatial Median for Robust Data Mining. <http://users.jyu.fi/~samiayr/pdf/ayramo_eurogen05.pdf>`_
 
+.. _huber_regression:
+
+Huber Regression
+----------------
+
+The :class:`HuberRegressor` is an extension of :class:`Ridge` except that
+the loss above a certain threshold is linear. It differs from :class:`TheilSenRegressor`
+and :class:`RANSACRegressor` because it does not ignore the effect of the outliers
+but gives a lesser weight to them.
+
+.. figure:: ../auto_examples/linear_model/images/plot_huber_vs_ridge_001.png
+   :target: ../auto_examples/linear_model/plot_huber_vs_ridge.html
+   :align: center
+   :scale: 50%
+
+The loss function that :class:`HuberRegressor` minimizes is given by
+
+.. math::
+
+   \underset{w}{min\,} {n_samples*\sigma + H_m(\frac{X w - y}{\sigma})*\sigma + \alpha {||w||_2}^2}
+
+where
+
+.. math:: H(z)
```

Comment:
H_m(z) to stay consistent with paper notation (eq. 5 subsection 2 has M, the eq above which only has H(z) has no M term). Also H_M above and here would be more consistent IMO